INTEGRATE.md

```diff
@@ -0,0 +1,33 @@
+# Local integration between Porta/System with Keycloak through Zync
+With these 3 components installed, you will be able to synchronize your applications in Porta with your Keycloak (Red Hat SSO)
```

Comment:
I would avoid using `you` in instructions but be more imperative and using the present tense

Proposition:

```suggestion
Install these 3 components to have a complete integration of managed SSO application in Porta
```